Assets/UI/Popups/greatpeoplepopup.lua

```diff
@@ -154,14 +154,13 @@ function ViewCurrent( data:table )
         textureOffsetY = 0;
         textureSheet = "GreatPeopleClass90";
       end
-      instance.ClassImage:SetTexture( textureOffsetX, textureOffsetY, textureSheet );
     end
 
     -- Grab icon of the great person themselves; first try a specific image, if it doesn't exist
     -- then grab a generic representation based on the class.
     --if (kPerson.ClassID ~= nil) and (kPerson.IndividualID ~= nil) then
       --local portrait:string = "ICON_" .. individualData.GreatPersonIndividualType;
-      --textureOffsetX, textureOffsetY, textureSheet = IconManager:FindIconAtlas(portrait, 160, true);
+      --textureOffsetX, textureOffsetY, textureSheet = IconManager:FindIconAtlas(portrait, 216, true);
       --if textureSheet == nil then   -- Use a default if none found
         --print("WARNING: Could not find icon atlas entry for the individual Great Person '"..portrait.."', using default instead.");
         --portrait = "ICON_GENERIC_" .. classData.GreatPersonClassType .. "_" .. individualData.Gender;
```

Comment:
I would uncomment this block but let the 
```
-- if (isValid) then
  --instance.BiographyPortrait:SetIcon(portrait);
--end
```
commented

Reply:
I left the `print("WARNING...");` commented. I remember you somewhere wrote, you want to see errors primarily and have a cleaner log.

Reply:
Indeed, thx